Code/Framework/AzQtComponents/AzQtComponents/Components/Widgets/FileDialog.h

```diff
@@ -24,7 +24,16 @@ namespace AzQtComponents
         static QString GetSaveFileName(QWidget* parent = nullptr, const QString& caption = QString(),
             const QString& dir = QString(), const QString& filter = QString(),
             QString* selectedFilter = nullptr, QFileDialog::Options options = QFileDialog::Options());
-        
+
+        //! Helper function which simply calls the GetSaveFileName function
+        static QString GetSaveFileName_(
+            QWidget* parent = nullptr,
+            const QString& caption = QString(),
+            const QString& dir = QString(),
+            const QString& filter = QString(),
+            QString* selectedFilter = nullptr,
+            QFileDialog::Options options = QFileDialog::Options());
+
```

Comment:
What is the point of having this function, if it all it does it is call the function without the underscore?

Reply:
You're right, it's no longer require3d. Removed.

Reply:
Thanks for the clarification.